File: api/src/main/java/com/jakedelivery/api/userorder/dto/response/UserOrderDetailResponse.java
package com.jakedelivery.api.userorder.dto.response;

import com.jakedelivery.api.store.dto.StoreResponse;
import com.jakedelivery.api.storemenu.dto.StoreMenuResponse;
import com.jakedelivery.api.userorder.dto.UserOrderResponse;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserOrderDetailResponse {
    private UserOrderResponse userOrderResponse;
    private StoreResponse storeResponse;
    private List<StoreMenuResponse> storeMenuResponses;
}

File: api/src/main/kotlin/com/jakedelivery/api/userorder/dto/UserOrderDetailResponse.kt
package com.jakedelivery.api.userorder.dto

import com.jakedelivery.api.store.dto.StoreResponse
import com.jakedelivery.api.storemenu.dto.StoreMenuResponse

data class UserOrderDetailResponse (
    val userOrderResponse: UserOrderResponse? = null,
    val storeResponse: StoreResponse? = null,
    val storeMenuResponses: List<StoreMenuResponse>? = null
)
